lib/cockpit-components-table.scss

```diff
@@ -1,5 +1,6 @@
-@import "~@patternfly/patternfly/components/Table/table.scss";
-@import "~@patternfly/patternfly/components/Table/table-grid.scss";
+@import "@patternfly/patternfly/base/patternfly-variables.scss";
```

Comment:
Hmm, I really thought it's a good idea to have patternfly-variables everywhere accessible. (no in podman but in cockpit) almost all our CSS files use PF4 variables. I am not sure if it makes sense to include in every single files the PF variables. We currently import the variables with sass-resource-loader but maybe I can find another solution.

Reply:
For example in this project here, you missed src/podman.scss, it also uses PF4 variables.

Reply:
Hmm, sass-loader (and sassc) complains loudly and bitterly if it does not know about a variable, so it seemed fairly complete to me (it takes quite a lot of effort in cockpit to get it to build, due to tracking down all the missing dependencies).

My gut feeling was that including the bits into page.scss is a good enough "global" place? Every page ought to import that, no?

Reply:
Right, we can do it like this. page.scss is global enough indeed.